newdocs/list_of_all_used_software.md

```diff
@@ -0,0 +1,41 @@
+| Tool                   | Version      | Type        | URL                                                    | External contact                                       | Internal contact                      | Short description                                                                         |
```

Comment:
This file is outdated, we need to transfer the version updates to the `docs` one